.travis.yml

```diff
@@ -35,7 +34,6 @@ env:
 before_install:
   - export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY
   - export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_KEY
-  - npm config set '//registry.npmjs.org/:_authToken' ${NPM_TOKEN}
```

Comment:
you don't need it since you don't use private `@typeform` packages (and shouldn't as it's OS)

Reply:
Very soon we will want to use Jarvis for deployments. Not as dependency, just installing it locally inside Travis job.
https://jira.typeform.tf/browse/DIST-242

Reply:
I see 🤔